src/index.js

```diff
@@ -16,46 +18,25 @@ const DEFAULT_ALLOW_HEADERS = [
   'Accept'
 ]
 
-const DEFAULT_MAX_AGE_SECONDS = 60 * 60 * 24 // 24 hours
-
 const cors = options => handler => (req, res, ...restArgs) => {
   const {
-    maxAge,
-    origin,
-    allowHeaders,
-    exposeHeaders,
-    allowMethods
+    origin = '*',
+    maxAge = DEFAULT_MAX_AGE_SECONDS,
+    allowMethods = DEFAULT_ALLOW_METHODS,
+    allowHeaders = DEFAULT_ALLOW_HEADERS,
+    exposeHeaders
   } = (options || {})
 
-  res.setHeader(
-    'Access-Control-Max-Age',
-    '' + (maxAge || DEFAULT_MAX_AGE_SECONDS)
-  )
-
-  res.setHeader(
-    'Access-Control-Allow-Origin',
-    (origin || '*')
-  )
-
-  res.setHeader(
-    'Access-Control-Allow-Methods',
-    (allowMethods || DEFAULT_ALLOW_METHODS).join(',')
-  )
-
-  res.setHeader(
-    'Access-Control-Allow-Headers',
-    (allowHeaders || DEFAULT_ALLOW_HEADERS).join(',')
-  )
+  res.setHeader('Access-Control-Allow-Credentials', 'true')
+  res.setHeader('Access-Control-Max-Age', maxAge)
+  res.setHeader('Access-Control-Allow-Origin', origin)
+  res.setHeader('Access-Control-Allow-Methods', allowMethods.join(','))
+  res.setHeader('Access-Control-Allow-Headers', allowHeaders.join(','))
 
-  if (exposeHeaders && exposeHeaders.length) {
-    res.setHeader(
-      'Access-Control-Expose-Headers',
-      exposeHeaders.join(',')
-    )
+  if (exposeHeaders instanceof Array) {
+    res.setHeader('Access-Control-Expose-Headers', exposeHeaders.join(','))
```

Comment:
Since we're defaulting `exposeHeaders` to an empty array this will always be true. What do you think about `if (exposeHeaders.length)`?